rimage/depth_map.go

```diff
@@ -117,41 +116,17 @@ func (dm *DepthMap) SubImage(r image.Rectangle) *DepthMap {
 // or if it can be converted into one.
 func ConvertImageToDepthMap(img image.Image) (*DepthMap, error) {
 	switch ii := img.(type) {
-	case *LazyEncodedImage:
-		decoded, err := decodeImage(
-			context.Background(), ii.RawData(), ii.MIMEType(), ii.Bounds().Dx(), ii.Bounds().Dy(), false)
-		if err != nil {
-			return nil, err
-		}
-		return ConvertImageToDepthMap(decoded)
 	case *DepthMap:
 		return ii, nil
 	case *imageWithDepth:
 		return ii.Depth, nil
 	case *image.Gray16:
 		return gray16ToDepthMap(ii), nil
-	case *image.NRGBA64:
```

Comment:
Is it okay to get rid of NRGBA64 before reintroducing .dat? CC @bhaney

Reply:
Right now, none of our camera servers return a raw depth image encoded in NRGBA64. Depth info is still successfully transmitted as a png (and I think SLAM still is just using png?) 

Given that our new header will only apply for 8-bit color too, I think the 16-bit color "encodings" will become broken if we leave it in. So it's best to remove it.

Reply:
Thank you!